tensorflow/compiler/xla/service/gpu/nvptx_compiler.cc

```diff
@@ -198,6 +198,42 @@ absl::optional<bool> CanShareBufferHint(const HloInstruction* user,
   return absl::nullopt;
 }
 
+// Try to load ptx from files defined in the FLAGS. If successful, return true.
+bool MaybeLoadPtxFromFile(const HloModule* module, std::string* ptx) {
```

Comment:
The diff show up stream. I didn't move MaybeLoadPtxFromFile.
I moved the function WarnIfBadDriverJITVersion outside the unnamed namespace.